packages/core/src/actions/send.ts

```diff
@@ -134,12 +134,32 @@ export function send<
           targetActorRef = state.children[resolvedTarget];
         }
         if (!targetActorRef) {
-          throw new Error(
-            `Unable to send event to actor '${resolvedTarget}' from machine '${state.machine.id}'.`
-          );
+          // throw new Error(
+          //   `Unable to send event to actor '${resolvedTarget}' from machine '${state.machine.id}'.`
+          // );
+          return [
+            state,
+            {
+              type: actionTypes.send,
+              params: {
+                ...params,
+                _event: resolvedEvent,
+                event: resolvedEvent.data,
+                internal: false,
+                to: undefined,
+                delay: undefined
+              },
+              execute: (actorCtx) => {
+                actorCtx.system.deadLetters.send({
+                  type: 'deadLetter',
+                  event: resolvedEvent
+                });
+              }
```

Comment:
could you specify the exact role of this dead letter thing? for instance, i don't think it should be send action's role to send it there. Why don't we put it in the `actor.send` method?